controllers/virtualmachine/storagepolicyusage/webhookconfiguration/webhookconfiguration_controller.go

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) 2024 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package webhookconfiguration
```

Comment:
Please relocate this under `./controllers/infra/validatingwebhookconfiguration`.